src/Microsoft.ML.Data/DataLoadSave/DataOperationsCatalog.cs

```diff
@@ -63,7 +63,7 @@ internal DataOperationsCatalog(IHostEnvironment env)
         /// results.
         /// </summary>
         /// <typeparam name="TRow">The user-defined item type.</typeparam>
-        /// <param name="data">The enumerable data containing type <typeparamref name="TRow"/> to convert to an<see cref="IDataView"/>.</param>
+        /// <param name="data">The enumerable data containing type <typeparamref name="TRow"/> to convert to a <see cref="IDataView"/>.</param>
```

Comment:
shouldn't it be "an IDataView"?

Reply:
It should be "... to convert to **a** Microsoft.ML.IDataView" as the `<see cref="IDataView"/> is printed as "Microsoft.ML.IDataView".

Reply:
Oh that's tricky... thanks for explanation